sdk/core/azure-core-amqp/src/samples/java/com/azure/core/amqp/models/AmqpAddressJavaDocCodeSamples.java

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp.models;
+
+/**
+ * Class contains sample code snippets that will be used in javadocs.
+ */
+public class AmqpAddressJavaDocCodeSamples {
```

Comment:
We should have a single class for all javadocs. Users landing on our samples directory will see too many javadoc classes and can distract them from real samples.